jsbits/time.js

```diff
@@ -1,4 +1,9 @@
+//#OPTIONS: CPP
+
 function h$js_futimes(fd,atime,mtime) {
+#ifdef GHCJS_BROWSER
+    throw "h$js_futimes unsupported";
+#else
   if (!h$isNode()) {
```

Comment:
Could it be 
```suggestion
  if (!h$isNode() || defined(GHCJS_BROWSER)) {
```
?

Reply:
Not sure what you mean, we want a preprocessor-level `if`, not a runtime-level `if` here.

Reply:
Fair enough.

Reply:
In other words if ghc is compiled with `-DGHCJS_BROWSER` then `h$isNode()` is undefined at runtime and we don't want to include this code at all.